Symfony/CS/Fixer/Symfony/SpacesAfterSemicolonFixer.php

```diff
@@ -27,15 +27,26 @@ public function fix(\SplFileInfo $file, $content)
     {
         $tokens = Tokens::fromCode($content);
 
-        for ($index = count($tokens) - 3; $index > 0; --$index) {
-            if (!$tokens[$index]->equals(';') || $tokens[$index + 2]->isComment()) {
+        for ($index = count($tokens) - 2; $index > 0; --$index) {
+            if (!$tokens[$index]->equals(';')) {
                 continue;
             }
 
             if (!$tokens[$index + 1]->isWhitespace()) {
-                $tokens->insertAt($index + 1, new Token(array(T_WHITESPACE, ' ')));
-            } elseif (!$tokens[$index + 1]->equals(array(T_WHITESPACE, ' ')) && $tokens[$index + 1]->isWhitespace(array('whitespaces' => " \t"))) {
-                $tokens[$index + 1]->setContent(' ');
+                if (!$tokens[$index + 1]->equals(')')) {
+                    $tokens->insertAt($index + 1, new Token(array(T_WHITESPACE, ' ')));
+                }
+            } elseif (
+                isset($tokens[$index + 2])
+                && !$tokens[$index + 1]->equals(array(T_WHITESPACE, ' '))
+                && $tokens[$index + 1]->isWhitespace(array('whitespaces' => " \t"))
+                && !$tokens[$index + 2]->isComment()
+            ) {
+                if ($tokens[$index + 2]->equals(')')) {
+                    $tokens[$index + 1]->clear();
```

Comment:
is this the job of this fixer? I'm not sure.
On 2.x it is also done by `ParenthesisFixer`..